.github/workflows/release.yml

```
@@ -0,0 +1,24 @@
+name: Release the gem
+
+on:
+  create:
+    ref_type: tag
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
```

Does it make sense to limit this to voxpupuli/beaker?
```suggestion
    runs-on: ubuntu-latest
    if: github.repository == 'voxpupuli/beaker'
```

---

Can't hurt!